### PR TITLE
Propose Storefront Design project

### PR DESCRIPTION
**Pull request by ryanchou-dev**
Thanks for making a PR! Please fill in the below:

Name: ryan

Do you run a club? 

nope! but i'm an active community member :DD

What relevant experiences do you have to help you write this Jam?

- wrote workshops for Hack Club before
- experience teaching Hack Club workshops + other CS stuff
~- undying love for figma~

Will you be available on Slack to provide support to Hack Clubbers who run into issues while going through your Jam?

yep!

### CODE REVIEW

**Comment by SerenityUX** [comment]
love how this is at the intersection of tech & entrepreneurship

**Review by Arpan-206 — APPROVED** [review]
About time, we **need** to have a design jam!

File: jams/Storefront_Design.md
Comment (miguelaenlle) [inline jams/Storefront_Design.md:1]: Love the concept of a Figma Jam! Design plays a huge role in the software development lifecycle, so it's great that members can gain exposure to it through this workshop. I want to run this next schoolyear at my school's Hack Club :) 

I left a suggestion about the Kahoots below, but everything else looks good.

File: jams/Storefront_Design.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+1. Exposes them to real-world design workflows
+2. Jammers are encouraged to make teams and work on their storefronts together (nothing better than peer-pressure!!)
+3. Seeing a finished prototype is really inspiring on getting to make it into a reality. Shows Jammers what the things they make can look like!
+
+- General outline of a club meet doing the Jam
+
+
+- **5m**: Setup, get everyone a Figma account and on a blank file
+- **10-15m**: Lecture on Figma's tools and features while making a storefront
+- (optional) **5m**: Kahoot on the things they just learned (for a prize?)
```
Comment (miguelaenlle) [inline jams/Storefront_Design.md]: I would omit the Kahoot part and go straight to building the storefronts, and then have workshop hosts help members live as they build. Kahoot setup would take longer than expected

**Review by miguelaenlle — APPROVED** [review]

File: jams/Storefront_Design.md (hunk 1 above)
Comment (ryanchou-dev) [reply jams/Storefront_Design.md]: makes sense! i was just tossing around ideas :>

**Review by faisalsayed10 — APPROVED** [review]
lgtm

**Comment by SerenityUX** [comment]
It would be so cool if this was two parts (one for designing the storefront & one for developing the site).

(keep in mind this would still be one bounty)
(you could use the Shopify API to create the storefront)

**Comment by ryanchou-dev** [comment]
ooh great idea! makes a lot of sense.


the only thing im worried about is having them learn css/html/js to implement it. i think it'd be a whole lot of content for the jammer (losing interest?), and im not sure i can finish it all by july 19th...

plus since the designs will be totally unique, it'd rly be up to the workshop hosts to do a lot of troubleshooting

i think something that might be better is that at the end, we add slides talking about how they can implement it, and give example implementations/resources if they want to try their hand at it.

**Comment by SerenityUX** [comment]
> ooh great idea! makes a lot of sense.
> 
> the only thing im worried about is having them learn css/html/js to implement it. i think it'd be a whole lot of content for the jammer (losing interest?), and im not sure i can finish it all by july 19th...
> 
> plus since the designs will be totally unique, it'd rly be up to the workshop hosts to do a lot of troubleshooting
> 
> i think something that might be better is that at the end, we add slides talking about how they can implement it, and give example implementations/resources if they want to try their hand at it.

It's okay to make prerequisites of web dev for part 2 (I'm actually working on a workshop that teaches those fundamental web dev skills). I would love for Hack Clubbers to make something that they (& others) actually use! Lots of high schoolers create successful Shopify businesses, and it would be so cool if your Jam could be a launching point for them.

**Comment by ryanchou-dev** [comment]
ohhh okay, yeah that sounds great!

**Review by sarthaktexas — COMMENTED** [review]
Do you think that the tools you are discussing are a bit advanced? I think you might benefit from just the basic tools.

**Comment by ryanchou-dev** [comment]
> Do you think that the tools you are discussing are a bit advanced? I think you might benefit from just the basic tools.

yeah! probably won't be talking about the tools i mentioned in the post. im working on an example storefront which doesn't use some of those tools...

will update the file when i get home 🔅

**Review by zeyuyaoy — APPROVED** [review]
RYANNNNN THIS LOOKS AWESOME 💯 💯 💯

**Comment by sahitid** [comment]
love it! lgtm

**Pull request merged**